examples/ffmpeg_video_decoding.cpp

```diff
@@ -0,0 +1,116 @@
+// The contents of this file are in the public domain. See LICENSE_FOR_EXAMPLE_PROGRAMS.txt
+/*
+
+    This is an example illustrating the use of the ffmpeg wrappers, 
+    in this case the decoding API.
+
+    This is a pretty simple example. It loads a raw codec file, parses chunks of 
+    data to the decoder and plots images to a GUI window.
+
+    Background about video files, something OpenCV doesn't make entirely clear:
```

Comment:
```suggestion
    Background about video files:
```
Just don't mention opencv :)